MonoGame.Framework/Content/ContentReaders/MultiArrayReader.cs

```diff
@@ -0,0 +1,108 @@
+﻿#region License
```

Comment:
Its a slightly modified copy of the old ArrayReader, so i didnt touch the license part. FIgured it was there for a reason.